editor/src/clj/editor/debug_view.clj

```diff
@@ -546,7 +548,7 @@
              (protobuf/bytes->map-with-defaults Lua$LuaModule))))
 
 (defn attach!
-  [debug-view project target build-artifacts]
+  [debug-view project target build-artifacts workspace]
```

Comment:
We could get the `workspace` using `(project/workspace project)`, so we don't need to add it as an argument here.

Reply:
thx! fixed